sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -27,7 +27,9 @@
 #include "net/gnrc/ipv6/nib.h"
 #include "net/gnrc/ipv6.h"
 #endif /* MODULE_GNRC_IPV6_NIB */
+#ifdef MODULE_GNRC_NETIF_PKTQ
```

Comment:
Here too
```suggestion
#if IS_USED(MODULE_GNRC_NETIF_PKTQ) 
```

Reply:
This is consistent with the other includes. Should I update the other as well?

Reply:
Maybe as own commit upfront, and the current commit is added afterwards? That way it would always be consistent and one commit per logical change.

Reply:
@miri64: Did as described above.